migrate.py

```diff
@@ -22,16 +23,21 @@
     auth0_mgmt_factory,
 )
 
-async def migrate_users():
+async def migrate_users(session):
+    print("Migrating Users...")
     async for user in auth0_manager.iter_users():
-        await insert_users(dict(user))
+        await add_user(session, dict(user))
 
-async def migrate_organization():
+async def migrate_organization(session):
+    print("Migrating Organizations...")
     org = await auth0_manager.get_organization()
-    await insert_org(dict(org))
+    await add_org(session, dict(org))
```

Comment:
I'm confused on this point -- when I run this and look in the database, the display name property is saved as `displayName` -- CAS should take care of that conversion. Are you sure there wasn't another one from another test you had in your db?

For example, if you had run this before when it was still using direct mongo (from PR 1) that likely was inserting a record with `display_name`. If you then ran this against the same Org ID, it wouldn't change that field because it's doing an upsert keyed off the org id. 

I'll test again, but if you have time to drop that org record and run the script again, that would be helpful!

Reply:
Also, thanks for testing the App in general! I was working on getting there yesterday but was hitting unrelated walls, so I'm glad you were able to test

Reply:
I was on `main` apologies!